L1Trigger/L1TTrackMatch/python/l1tTrackJetsEmulation_cfi.py

```diff
@@ -1,59 +1,45 @@
 import FWCore.ParameterSet.Config as cms
 
-l1tTrackJetsEmulation = cms.EDProducer('L1TrackJetEmulationProducer',
+l1tTrackJetsEmulation = cms.EDProducer('L1TrackJetEmulatorProducer',
 	L1TrackInputTag= cms.InputTag("l1tGTTInputProducer", "Level1TTTracksConverted"),
-        VertexInputTag=cms.InputTag("l1tVertexFinderEmulator", "l1verticesEmulation"),
-	MaxDzTrackPV = cms.double(0.5),
+        L1PVertexInputTag=cms.InputTag("l1tVertexFinderEmulator","l1verticesEmulation"),
+        MaxDzTrackPV = cms.double(1.0),
         trk_zMax = cms.double (15.) ,    # maximum track z
 	trk_ptMax = cms.double(200.),    # maximumum track pT before saturation [GeV]
 	trk_ptMin = cms.double(2.0),     # minimum track pt [GeV]
    	trk_etaMax = cms.double(2.4),    # maximum track eta
-	trk_chi2dofMax=cms.double(10.),	 # maximum track chi2/dof
-	trk_bendChi2Max=cms.double(2.2), # maximum track bendchi2
+        nStubs4PromptChi2=cms.double(10.0), #Prompt track quality flags for loose/tight
+        nStubs4PromptBend=cms.double(2.2),
+        nStubs5PromptChi2=cms.double(10.0),
+        nStubs5PromptBend=cms.double(2.2),
 	trk_nPSStubMin=cms.int32(-1),    # minimum PS stubs, -1 means no cut
-	minTrkJetpT=cms.double(5.),      # minimum track pt to be considered for track jet
+	minTrkJetpT=cms.double(-1.),      # minimum track pt to be considered for track jet
 	etaBins=cms.int32(24),
 	phiBins=cms.int32(27),
 	zBins=cms.int32(1),
-	d0_cutNStubs4=cms.double(0.15),
-	d0_cutNStubs5=cms.double(0.5),
+        d0_cutNStubs4=cms.double(-1),
+        d0_cutNStubs5=cms.double(-1),
 	lowpTJetMinTrackMultiplicity=cms.int32(2),
-        lowpTJetMinpT=cms.double(50.),
+        lowpTJetThreshold=cms.double(50.),
 	highpTJetMinTrackMultiplicity=cms.int32(3),
-        highpTJetMinpT=cms.double(100.),
+        highpTJetThreshold=cms.double(100.),
 	displaced=cms.bool(False), #Flag for displaced tracks
 	nStubs4DisplacedChi2=cms.double(5.0), #Displaced track quality flags for loose/tight
-	nStubs4Displacedbend=cms.double(1.7),
+	nStubs4DisplacedBend=cms.double(1.7),
 	nStubs5DisplacedChi2=cms.double(2.75),
-	nStubs5Displacedbend=cms.double(3.5),
+	nStubs5DisplacedBend=cms.double(3.5),
 	nDisplacedTracks=cms.int32(2) #Number of displaced tracks required per jet
 )
 
-l1tTrackJetsExtendedEmulation = cms.EDProducer('L1TrackJetEmulationProducer',
+l1tTrackJetsExtendedEmulation = l1tTrackJetsEmulation.clone(
 	L1TrackInputTag= cms.InputTag("l1tGTTInputProducerExtended", "Level1TTTracksExtendedConverted"),
```

Comment:
```suggestion
	L1TrackInputTag= ("l1tGTTInputProducerExtended", "Level1TTTracksExtendedConverted"),
```
and remove as well all other type specifications for all cloned parameters

Reply:
removed